projects/core/tokens/is-mobile-resolution.ts

```diff
@@ -16,7 +16,7 @@ export const TUI_IS_MOBILE_RES = new InjectionToken<Observable<boolean>>(
             return typedFromEvent(windowRef, 'resize').pipe(
                 share(),
                 startWith(null),
-                map(() => windowRef.innerWidth < mobile),
+                map(() => getViewportWidthBy(windowRef) < mobile),
```

Comment:
Should we use this util in other places ?
<img width="1407" alt="Screenshot 2022-06-22 at 11 15 59" src="https://user-images.githubusercontent.com/35179038/174979424-9e3acf25-5dd0-4202-904b-94e00364b7ba.png">

Reply:
@waterplea what do you think?

Reply:
Yes, makes sense to use it everywhere

Reply:
fixed